pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/copy/CopyExecutor.java

```diff
@@ -56,13 +60,32 @@ public void execute(Handle handle, long operatorId, long participantId, CopyConf
             CopyLocation target = pair.getTarget();
 
             if (source.getType() == CopyLocationType.ANSWER && target.getType() == CopyLocationType.ANSWER) {
+                QuestionDto sourceQuestion;
+                FormResponse sourceInstance;
                 String sourceStableId = ((CopyAnswerLocation) source).getQuestionStableId();
-                QuestionDto sourceQuestion = questionDtosByStableId.get(sourceStableId);
-                if (sourceQuestion == null) {
-                    continue; // Question might have been removed from activity, so we skip it.
+                String user = ((CopyAnswerLocation) source).getUser();
+                if (UserType.OPERATOR.equalsIgnoreCase(user)) {
+                    Stream<Governance> governances = handle.attach(UserGovernanceDao.class)
+                            .findActiveGovernancesByParticipantAndStudyIds(participantId, config.getStudyId());
+                    Optional<Governance> optionalGovernance = governances.findFirst();
+                    if (optionalGovernance.isEmpty()) {
+                        throw new DDPException(String.format("Governance not found for participant %s", participantId));
+                    }
+                    Governance governance = optionalGovernance.get();
+                    Map<Long, FormResponse> proxyActivities = retrieveActivityData(handle, governance.getProxyUserId(),
+                            List.copyOf(questionDtosByStableId.values()));
+                    sourceQuestion = questionDtosByStableId.get(sourceStableId);
+                    if (sourceQuestion == null) {
+                        continue; // Question might have been removed from activity, so we skip it.
+                    }
+                    sourceInstance = proxyActivities.get(sourceQuestion.getActivityId());
+                } else {
```

Comment:
@ssettipalli yes, all the existing copy configurations will have the same behavior like before. Just one if statement is added in case of operator which will be present only for lms and osteo.